README.md

```diff
@@ -21,7 +21,6 @@
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you put a logo of your own name (Not Microverse Logo) at the top of your readme file to present yourself as a brand in the job market 👍